core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -337,13 +337,17 @@ public void configureProperties() throws IOException {
   public abstract void close() throws IOException;
 
   /**
-   * Creates a file in the under file system with the indicated name.
+   * Creates a file in the under file system with the indicated name. This operation is expected
```

Comment:
I think "This operation is expected to be atomic" might be confusing. Does that mean this method, or the logical operation of creating, writing, and closing the file?